docs/docs/sourcing-from-gentics-mesh.md

```diff
@@ -0,0 +1,198 @@
+
+From the root of your project, run the development environment with `npm run develop`. Once the server has started, you should be able to open the following URL in your browser:
+
+`http://localhost:8000/___graphql`
+
+This will show you an interface where you can test your new content GraphQL API.
+
+Try running this query:
+
+```graphql
+query {
+  mesh {
+    nodes(filter: { schema: { is: vehicle } }, perPage: 2, page: 5) {
+      elements {
+        uuid
+        ... on MESH_vehicle {
+          fields {
+            slug
+            weight
+            vehicleImage {
+              path
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+Again, if everything is working properly, you should see a successful response in the shape of:
+
+```json
+{
+  "data": {
+    "mesh": {
+      "nodes": {
+        "elements": [
+          {
+            "uuid": "a5d81285b4884df1981285b488adf1b5",
+            "fields": {
+              "slug": "embraer-legacy-600",
+              "weight": 16000,
+              "vehicleImage": {
+                "path": "/images/embraer-legacy-600.jpg"
+              }
+            }
+          },
+          {
+            "uuid": "adaf48da8c124049af48da8c12a0493e",
+            "fields": {
+              "slug": "delorean-dmc-12",
+              "weight": 1230,
+              "vehicleImage": {
+                "path": "/images/delorean-dmc-12.jpg"
+              }
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+[You can read more about how to use the GraphQL API in the docs.](https://getmesh.io/docs/graphql/)
+
+### Getting content on the page
+
+For the purpose of this guide I've removed all the layout, SEO, link or other components in the Gatsby starter `index.js` file. Open the index file located at `src/pages/index.js` and replace the content with this code:
+
+```javascript:title=src/pages/index.js
+import React from "react"
+import { StaticQuery } from "gatsby"
+
+const IndexPage = () => (
+  <StaticQuery
+    query={graphql`
+      query {
+        mesh {
+          nodes(filter: { schema: { is: vehicle } }) {
+            elements {
+              uuid
+              ... on MESH_vehicle {
+                fields {
+                  name
+                  weight
+                }
+              }
+            }
+          }
+        }
+      }
+    `}
+    render={data => (
+      <div>
+        <h1>Vehicles</h1>
+        <ul>
+          {data.mesh.nodes.elements.map(vehicle => {
+            const { name, weight } = vehicle.fields
+            return (
+              <li>
+                <strong>{name}: </strong>
+                {weight} kg
+              </li>
+            )
+          })}
+        </ul>
+      </div>
+    )}
+  />
+)
+
+export default IndexPage
+```
+
+Once saved you can access the page via `http://localhost:8000`
+
+You now have:
+
+1. Added the `StaticQuery` component to a page that allows you to fetch content from the GraphQL API.
```

Comment:
whats more suggested? the `useStaticQuery` or `StaticQuery`? i personally would prefer the hook

Reply:
I also prefer usage of hooks. The examples in the guides on static-query do however only show usage of `StaticQuery`. It might be better to keep docs consistent?

https://www.gatsbyjs.org/docs/static-query/